components/org.wso2.carbon.identity.recovery/src/main/java/org/wso2/carbon/identity/recovery/internal/service/impl/UserAccountRecoveryManager.java

```diff
@@ -553,6 +602,29 @@ Some claims (Eg:- Birth date) can have "/" in claim values. But in user store le
         }
     }
 
+    /**
+     * Get all the domain names related to user stores.
+     * @param tenantId   Tenant ID
+     * @return A list of all the available domain names for given tenant
+     */
+    private String[] getDomainNames(int tenantId) throws IdentityRecoveryServerException {
```

Comment:
Can't we return a list here?
(Any reason to change type from list to array)

Reply:
Yes, we can return a list here

Reply:
Resolved by https://github.com/wso2-extensions/identity-governance/pull/623/commits/b7d9ed7ba946017669ed14519cf195b5ff039fce